index.html

```diff
@@ -10,13 +10,17 @@
 </head>
 <body>
     <header class="header">
-        <nav class="navbar" >
+        <nav class="navbar">
             <a class="logo" href="#">My Logo</a>
-            <i class="fa-solid fa-bars"></i>
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
             <ul class="menu">
-                <li class="list"><a href="">Portfolio</a></li>
-                <li class="list"><a href="">About</a></li>
-                <li class="list"><a href="">Contact</a></li>
+                <li class="list"><a href="#">Portfolio</a></li>
+                <li class="list"><a href="#">About</a></li>
+                <li class="list"><a href="#">Contact</a></li>
```

Comment:
- [ ] kindly consider giving these nav links the same font size as given in the Figma design to meet the below design requirements
> The hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins).The hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins).